pkg/api/types.go

```diff
@@ -93,6 +93,9 @@ var (
 
 // InspectorMetadata is the metadata type with information about image-inspector's operation
 type InspectorMetadata struct {
+	ImageAcquireSuccess bool   // status of aquiring the image
```

Comment:
@enoodle do you need to make this an attribute or could it be just a method that that relies on whether `ImageAcquireError` is `nil` or not?

Reply:
I can do that, It is mostly effecting on the user side.